targets/XPlatCppSdk/source/build/Xbox/TestXboxApp/TestXboxApp.vcxproj.ejs

```diff
@@ -104,18 +107,26 @@
     </Link>
     <ClCompile>
       <PrecompiledHeader>Use</PrecompiledHeader>
-      <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
+      <PrecompiledHeaderFile>TestAppPch.h</PrecompiledHeaderFile>
       <AdditionalUsingDirectories>$(Console_SdkPackagesRoot);$(Console_SdkWindowsMetadataPath);%(AdditionalUsingDirectories)</AdditionalUsingDirectories>
       <Optimization>MaxSpeed</Optimization>
       <PreprocessorDefinitions>NDEBUG;__WRL_NO_DEFAULT_LIB__;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <WarningLevel>Level4</WarningLevel>
       <FunctionLevelLinking>true</FunctionLevelLinking>
       <IntrinsicFunctions>true</IntrinsicFunctions>
-      <AdditionalIncludeDirectories>$(SdkSourceDir)\include;$(ExternalDir)\jsoncpp\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>$(TestSourceDir)\TestApp;$(SdkSourceDir)\include;$(ExternalDir)\jsoncpp\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
     </ClCompile>
     <FXCompile>
       <ShaderModel>5.0</ShaderModel>
     </FXCompile>
+    <CustomBuildStep>
+      <Command>copy $(PF_TEST_TITLE_DATA_JSON) $(Platform)\Layout\Image\Loose /Y</Command>
+    </CustomBuildStep>
+    <CustomBuildStep>
+      <Message>Copying testTitleData.json for packaging</Message>
+      <TreatOutputAsContent>true</TreatOutputAsContent>
+      <Outputs>$(Platform)\Layout\Image\Loose\testTitleData.json</Outputs>
```

Comment:
I would like to add a new .gitignore at exactly this location, which excludes:
```**/*testTitleData*```
And add that to SdkGenerator.
(We recently had a security issue, so I just want to be extra paranoid)